README.md

```diff
@@ -83,18 +83,19 @@ supported versions of Kubernetes clusters.
 
 #### Compatibility matrix
 
-|                    | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 | Kubernetes 1.8 | Kubernetes 1.9 | Kubernetes 1.10 | Kubernetes 1.11 | Kubernetes 1.12 | Kubernetes 1.13 |
-|--------------------|----------------|----------------|----------------|----------------|----------------|----------------|-----------------|-----------------|-----------------|-----------------|
-| client-python 1.0  | +              | ✓              | -              | -              |-               |-               |                 |                 |                 |                 |
```

Comment:
should 1.4 be removed, as it is not compatible at all

Reply:
I agree. Updated